src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -0,0 +1,51 @@
+import classNames from 'classnames';
+import React from 'react';
+import { StatusToFilter } from '../../types/StatusToFilter';
+
+interface Props {
+  filterStatus: string,
+  onFilterStatusChange: (newStatus: StatusToFilter) => void,
+}
+
+export const TodoFilter: React.FC<Props> = React.memo(({
+  filterStatus, onFilterStatusChange,
+}) => {
+  return (
+    <nav className="filter" data-cy="Filter">
+      <a
+        data-cy="FilterLinkAll"
+        href="#/"
+        className={classNames(
+          'filter__link',
+          { selected: filterStatus === StatusToFilter.All },
+        )}
+        onClick={() => onFilterStatusChange(StatusToFilter.All)}
+      >
+        {StatusToFilter.All}
+      </a>
+
+      <a
+        data-cy="FilterLinkActive"
+        href="#/active"
+        className={classNames(
+          'filter__link',
+          { selected: filterStatus === StatusToFilter.Active },
+        )}
+        onClick={() => onFilterStatusChange(StatusToFilter.Active)}
+      >
+        {StatusToFilter.Active}
+      </a>
```

Comment:
you have three equal tags, so it will be advisable to make one component with such tag that accepts different props. It will be more understandable that it is not just tags but tags with one logic